src/base/interfaces.ts

```diff
@@ -11,9 +11,38 @@ import {AppHeader} from 'phovea_ui';
 
 
 export interface IAdditionalColumnDesc extends IColumnDesc {
+  /**
+   * used internally to match selections to column
+   * @default -1
+   */
   selectedId: number;
+  /**
+   * used internally to match selections to multiple columns
+   * @default: undefined
+   */
   selectedSubtype?: string;
+  /**
+   * part of the initial ranking by default
+   * @default true
+   */
   initialRanking: boolean;
+  /**
+   * adds the item to a parent group
+   */
+  group?: {
```

Comment:
I think `group` in the context of the ColumnDesc is a bit misleading, as columns can be grouped and there is also a [GroupColumn](https://github.com/lineupjs/lineupjs/blob/master/src/model/GroupColumn.ts) . Can we rename this to something more precise that it is used for the add column dialog drop down?

Reply:
@dvdanielamoitzi @lehnerchristian @puehringer Any ideas?

Reply:
@thinkh @dvdanielamoitzi @lehnerchristian How about `chooserGroup`, as the `Add Column` dialog is internally called column chooser?

Reply:
@thinkh and I agreed that `chooserGroup` is ok!